asf_core_data/getters/data_getters.py

```diff
@@ -229,11 +228,26 @@ def load_s3_data(
     """
 
     if fnmatch(file_name, "*.xlsx"):
-        data = pd.read_excel(
-            os.path.join("s3://" + bucket_name, file_name),
-            sheet_name=None,
-            dtype=dtype,
-        )
+        excel_file = pd.ExcelFile(os.path.join("s3://" + bucket_name, file_name))
+        sheet_names = excel_file.sheet_names
+
+        # if excel file has data in multiple sheets
+        if len(sheet_names) > 1:
+            data = pd.DataFrame()
+            for sheet in sheet_names:
+                aux = pd.read_excel(
+                    os.path.join("s3://" + bucket_name, file_name),
+                    sheet_name=sheet,
+                    dtype=dtype,
+                )
+                data = pd.concat([data, aux])
+                del aux
+        else:  # only one sheet
+            data = pd.read_excel(
+                os.path.join("s3://" + bucket_name, file_name),
+                sheet_name=None,
+                dtype=dtype,
+            )
         if len(data) > 1:
             return data
         else:
```

Comment:
```suggestion
        data = pd.read_excel(
            os.path.join("s3://" + bucket_name, file_name),
            sheet_name=None,
            dtype=dtypes,
        )
        if len(data) > 1:
            if isinstance(data, dict) and (len(data.keys()) > 1):
                return pd.concat(
                    data,
                    ignore_index=True,
                )
            else:
                return data
        else:
            return data[list(data.keys())[0]]
```

Reply:
hey @sqr00t , when you reviewed this file I had not committed all the necessary changes. When you're back from leave, can you please check that the most up to date version of this script works for you? For a bit more context, the changes to this file should fix issue #54